client/core/core.go

```diff
@@ -1420,12 +1420,12 @@ func (c *Core) connectAndUpdateWallet(w *xcWallet) error {
 	addr := w.currentDepositAddress()
 	newAddr, err := c.connectWallet(w)
 	if err != nil {
-		return err // core.Error with code connectWalletErr
+		return fmt.Errorf("connectWallet: %w", err) // core.Error with code connectWalletErr
 	}
 	if newAddr != addr {
 		c.log.Infof("New deposit address for %v wallet: %v", unbip(assetID), newAddr)
 		if err = c.storeDepositAddress(w.dbID, newAddr); err != nil {
-			return err
+			return fmt.Errorf("storeDepositAddress: %w", err)
```

Comment:
trying to debug wallet connect timeouts a bit with these.  I'm pretty sure it's just trying to open another wallet instance using the same btcwallet db though